plugins/janus_audiobridge.c

```diff
@@ -3227,7 +3330,7 @@ static void *janus_audiobridge_handler(void *data) {
 					opus_encoder_ctl(participant->encoder, OPUS_SET_MAX_BANDWIDTH(OPUS_BANDWIDTH_WIDEBAND));
 				}
 				/* FIXME This settings should be configurable */
```

Comment:
I think we can remove the FIXME comment above now.

Reply:
Ok